### ISSUE TITLE
Edit translation for custom entity with AdminList

### ISSUE THREAD

**Issue by cmberlin** [Kunstmaan/KunstmaanBundlesCMS#401]
Hi
First of all thanks for all the work you have done so far.
Is there an easy way to handle translation for fields (handled with Gedmo\Translatable) of custom entities edited through AdminListBundle? By now I am trying to add extra item actions to get it done.

Thanks
Carsten

**Comment by Perni1984** [comment]
Hi Carsten, how did you solve it - I am facing the same situation here.

**Comment by cmberlin** [comment]
Hi Christoph,
you can switch language on the top right navigation to edit translation of entities.

**Comment by Perni1984** [comment]
Hi @cmberlin, thanks for letting me know. However, when I did that, my adminlist entities where the same for each language, so the language setting didn't really have any impact (I was expecting that I have "shadow" copies of each object and can overwrite certain fields for certain languages).

To achieve that was fairly simple although. What I did, was to '''use  Gedmo\Mapping\Annotation as Gedmo; ''' and put an '''@Gedmo\Translatable''' Annotation on the fields I wanted to have language specific. Now all the fields that are not translateable are the same for each language, and the other fields can be changed per language.
